### ISSUE TITLE
Cannot see files in folders since Container Manager feature

### ISSUE THREAD

**Issue by walawa75** [jlesage/docker-crashplan-pro#438]
### Current Behavior

Hello

Since the appearance of Container Manager instead of Docker package I noticed that the backup was constantly stuck at 0 files found (0mb) which is not normal. So I browsed through files and noticed that folders and files were not visible. The permissions are good (USER_ID=0 and GROUP_ID=0).
The problem happened after a reboot following an image update.
I noticed that by restarting the container from the command line, the problem was gone.
Does anyone also have the problem?

What to do to continue to benefit from the evolutions of Container Manager (simple update feature) without having to restart systematically on the command line.

I specify that the bug is in no way related to the good work of our dear JLesage ;-)

### Environment

- OS:
- OS version:
- CPU:
- Docker version:
- Device model:
- Browser/OS:

### Container creation

Container Manager package

### Container log

```text
no abnormal log
```

**Comment by jlesage** [comment]
I assume you are using a Synology NAS?
Looks like the issue where `/volume1` is mapped to `/storage` (see #310).  Is it the case?

**Comment by walawa75** [comment]
Yes I use a Synology NAS but the issue not seems to be the same as #310 tab.
I will try to make new tests later

**Comment by jlesage** [comment]
You can also share the output of `docker inspect <container name>` so we can have a complete picture of the container's configuration.

**Comment by walawa75** [comment]
I deleted the container and created a new one. Issue seems gone when container started from Container Manager

**Comment by walawa75** [comment]
Hello,
Now I get 1,5 years to backup 4 TB. How to increase backup speed ? My ISP speed is about 700 mbps but crashplan backup is really slow.

**Comment by jlesage** [comment]
Yes, it is known that upload speed is slow.  However, because of deduplication, a lot of data don't really have be uploaded.  If you look at Tools->History, you can see the "Effective rate": the value can be very high and would typically improve over time.

**Comment by Bagu** [comment]
I had the same problem.
In fact, when the docker container was created (a long time ago), the storage volume was not requested.
Since the last update, not only is it requested, but if it hadn't been configured originally, it points to an empty virtual folder.
I installed portainer to change the configuration and set storage bind to volume1.
Since then, I've been able to browse storage, but for crashplan, it's not the same folder structure. As a result, the backup started from scratch. I hope that deduplication will play its part, otherwise I'm in for several months of backups.
